plotting/ft_plot_cloud.m

```diff
@@ -0,0 +1,137 @@
+function ft_plot_cloud(pos, varargin)
+
+% FT_PLOT_CLOUD visualizes iEEG data as a spherical cloud
+%
+% Use as
+%   ft_plot_cloud(pos, 'funparam', dat, ...)
+% where the first argument is the sensor array as returned by FT_ELECTRODEPLACEMENT,
+% FT_READ_SENS or FT_PREPARE_VOL_SENS.
+%
+% Optional input arguments should come in key-value pairs and can include
+%   'funparam'        = Nx1 array where N is the number of sensors, used to
```

Comment:
this should be passed in another way, more like ft_plot_topo and ft_plot_topo3. I'll change that after the merge of this version.